ansible_builder/cli.py

```diff
@@ -34,8 +52,10 @@ def run():
     ab = prepare()
 
     print('Processing...', end='\r')
+    # TO DO: stdout that prints in real-time and/or a "spinner"
```

Comment:
Can we make these all `TODO`? A lot of editors look for this token, and I dont think they are smart enough to find "TO DO" as separate words.